public/manifest.json

```diff
@@ -1,25 +1,20 @@
 {
-  "short_name": "React App",
```

Comment:
all json files according to convention should have double quotes, right now it is invalid json